src/internals/utils.js

```diff
@@ -103,16 +118,41 @@ export function processTheme(theme) {
   return { definition, values };
 }
 
+const THEME_PRIORITIZED_KEYS = [
```

Comment:
There is no need for priority ordering for these keys - they can be in any order. This means that you can derive them from the `THEME_VALUES` object from `constants.ts`:

```js
const THEME_KEYS = Object.keys(THEME_VALUES);
```